### PR TITLE
Add support for R in documentation.  

### PR DESCRIPTION
**Pull request by kanielc**
Also fix a bug for reading credentials in DefaultSource

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,24 @@ sqlContext.sql("CREATE TEMPORARY TABLE students_table USING com.stratio.provider
 sqlContext.sql("SELECT * FROM students_table").collect()
 
 ```
+
+### R API
+
```
Comment (pmadrigal) [inline README.md]: Hi, first of all, thank you very much!

We don't know what environment are you using (Rstudio, sparkR shell, etc) and we need to show how to set the environment in the documentation, all the step for run the example correctly.

**Comment by kanielc** [comment]
I did it in both environments, but I'll clean it up to demonstrate from the sparkR shell.

**Comment by kanielc** [comment]
Alright have a look now and let me know what you think.

To do the same from RStudio requires bringing in the libraries and creating the SparkContext manually.  I primarily use the shell though right now.

**Comment by pmadrigal** [comment]
Now is perfect! thank you very much!

**Pull request merged**